### DISCUSSION TITLE
Academic Reverification Required

### DISCUSSION THREAD

**Discussion by PingWang28** [community/community#122409] [GitHub Education]
### Hi everyone,

Hello:
I'm a teacher , I  joined GitHub Education in April 15th . I remember I selected as teacher . But now I get a mail from  GitHub Education Team let me reverify my information again .  I follow the step in the mail , explore the link  https://education.github.com/benefits  and click the "submit your information again."   But  two events are strange , one is that it seems I was recongized as a student , the second is when I click the "submit your information again."  it always jump to the link https://github.com/pricing  , I have no way to fill my information again .  What can I do for this ?

**Comment by salma-elbakkouri** [comment]
since the link is redirecting you to the pricing page instead of allowing you to resubmit your verification i think this could be a technical glitch or bug, About the **Account type** i think  there was a mix-up with how your account was categorized and you should try to directly contact GitHub Support. You can reach out to them through the GitHub Education support page or the contact link typically found at the bottom of the GitHub Education site

**Reply by PingWang28** [reply]
yes, I also think this should be a bug , and I have try to get support from GitHub , but seems they ban this topic to open a ticket . I have to come here to get some support

**Comment by PingWang28** [comment]
@samus-aran , could you check my issue ?  Github need me reverify my information in 30 days, it has passed half of it . Thank you
